### PR TITLE
Slimes now don't rot again

### PR DESCRIPTION
**Pull request by Hmeister-fake**
## About the PR
 Slimes now shouldn't rot.

## Why / Balance
 As they were before the recent change to rotting, slimes had very little to work with and this at least gave them an upperhand comparable to Arachnid pockets and Reptilian tail grabbing. With rotting, slimes are just squishier and easier to kill rainbow urist mchands who OD faster from meds and i guess have funny blood?

## Technical details
 Added one line of code, someone please check if it's placed in the right area.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
 Literally one line of code

**Changelog**
 🆑 Hmeister

- tweak: Slimes don't Rot again.

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
Please test your changes before you make PRs

**Comment by Hmeister-fake** [comment]
> Please test your changes before you make PRs

can't, dev enivornment is bugged and i cant figure out how to fix it

**Comment by Kadeo64** [comment]
imo "slimes don't rot" seems like a bandaid change for slimes as a species when they could really use actual unique abilities, like [23425] attempts to do.

**Comment by Hmeister-fake** [comment]
> imo "slimes don't rot" seems like a bandaid change for slimes as a species when they could really use actual unique abilities, like [23425] attempts to do.

Yes, but untill then they shouldn't be almost objectively worse than Diona (lmao)

**Comment by Hmeister-fake** [comment]
> Please test your changes before you make PRs

it should work however, got help from a maintainer to confirm it.

**Comment by Nairodian** [comment]
Okay to properly explain why this isn't going to work. `noRot` is actually short form for no rotation, it's used for sprites that shouldn't rotate. You're looking for the component `- type: Perishable`. Please keep in mind if you're going to remove it from slimes, you're going to have to remove the perishable component from the base mob, and re-add it to every species except slimes.

**Comment by Just-a-Unity-Dev** [comment]
probably shouldn't make a bandaid when the proper PR is just a few days away - this will probably conflict with my pr too

File: Resources/Prototypes/Entities/Mobs/Species/slime.yml [hunk 1]
```diff
@@ -80,6 +80,7 @@
 - type: entity
   parent: MobHumanDummy
   id: MobSlimePersonDummy
+  noRot: true
```
Comment (DrMelon) [inline Resources/Prototypes/Entities/Mobs/Species/slime.yml:83]: See: https://github.com/space-wizards/space-station-14/pull/23452#issuecomment-1876120863

**Review by DrMelon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**